meson.build

```diff
@@ -6,9 +6,15 @@ link_args = []
 if cc.get_id() == 'msvc'
   add_project_arguments('-D_CRT_SECURE_NO_DEPRECATE',
     '-D_CRT_NONSTDC_NO_DEPRECATE', language : 'c')
-elif cc.get_id() == 'gcc' and host_machine.system() != 'windows'
-  vflag = '-Wl,--version-script,@0@/zlib.map'.format(meson.current_source_dir())
-  link_args += [vflag]
+else
+  # Don't spam consumers of this wrap with these warnings
```

Comment:
Maybe we just disable warnings as a target option via `override_options`?

Reply:
Not sure I understand what you mean. Do you mean `warning_level:` on the `library()` target? That's not the only way these warnings can be enabled, so this method would be more fool-proof.